content/general/concepts/data-structures/terms/algorithms/searching/terms/binary-search/binary-search.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: 'Binary Search Algorithm'
+Description: 'A searching algorithm to search an element in a sorted collection by dividing the collection in two halves.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Algorithms'
+  - 'Data Structures'
+  - 'Search Algorithms'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+**Binary Search** is an algorithm for searching an element within a sorted collection of items, primarily implemented with [arrays](https://www.codecademy.com/resources/docs/general/data-structures/array) or lists. The binary search algorithm follows a divide-and-conquer approach to determine whether the target element matches the collection's middle element. If there is a match, it provides the index of the middle element; otherwise, it proceeds to either side of the array, depending on the current comparison result.
```

Comment:
"The binary search algorithm follows a divide-and-conquer approach to determine whether the target element matches the collection's middle element"

This statement sounds like the binary search algorithm is checking to see whether the middle element is the target element in a collection. Can you reword this statement with a different definition of binary search?